harness/features/multithreading.local.test.ts

```diff
@@ -0,0 +1,242 @@
+import {
+    describeCapability,
+    forEachSDK,
+    getPlatformBySdkName,
+    LocalTestClient,
+    waitForRequest
+} from '../helpers'
+import { Capabilities } from '../types'
+import { getServerScope } from '../nock'
+import { config } from '../mockData'
+import { VariableType } from '@devcycle/types'
+import { optionalEventFields, optionalUserEventFields } from '../mockData/events'
+
+const scope = getServerScope()
+
+const expectedVariable = {
+    key: 'string-var',
+    defaultValue: 'default_value',
+    variationOn: 'string',
+    variableType: 'String',
+}
+
+const variableType = VariableType.string
+
+const bucketedEventMetadata = {
+    _feature: "638680d6fcb67b96878d90e6",
+    _variation: "638680d6fcb67b96878d90ec"
+}
+
+describe('Multithreading Tests', () => {
+    forEachSDK((name) => {
+        const expectedPlatform = getPlatformBySdkName(name, true)
+
+        describeCapability(name, Capabilities.local)(name, () => {
```

Comment:
This should probably be `Capabilities.multithreading`

Reply:
oops